scipy/stats/_multivariate.py

```diff
@@ -2642,3 +2644,143 @@ def entropy(self):
     method_frozen.__doc__ = doccer.docformat(
         method.__doc__, wishart_docdict_noparams)
     method.__doc__ = doccer.docformat(method.__doc__, wishart_docdict_params)
+
+class special_ortho_group_gen(multi_rv_generic):
+    r"""
+    A matrix-valued SO(N) random variable.
+
+    The `dim` keyword specifies the dimension N.
+
+    Methods
+    -------
+    ``rvs(dim=None, size=1, random_state=None)``
+        Draw random samples from SO(N).
+
+    Parameters
+    ----------
+    dim : scalar
+        Dimension of matrices
+
+    TODO: frozen object not implemented yet.
```

Comment:
leftover?
BTW, I'm not sure if a frozen version is *that* useful. If you've implemented it already, fine.